server/controllers/temporary-accommodation/manage/premisesController.test.ts

```diff
@@ -75,42 +86,25 @@ describe('PremisesController', () => {
     })
     ;(extractCallConfig as jest.MockedFn<typeof extractCallConfig>).mockReturnValue(callConfig)
     assessmentService.findAssessment.mockResolvedValue(assessment)
+
+    jest.spyOn(premisesUtils, 'tableRows')
+    jest.spyOn(premisesUtils, 'summaryList')
   })
 
   describe('index', () => {
-    const premisesRow1 = [
```

Comment:
`tableRows` is covered in its own test suite, so no need for this fixture data here.